Mixpanel/Mixpanel.m

```diff
@@ -50,11 +50,13 @@ @interface Mixpanel () <UIAlertViewDelegate>
 #endif
 {
     NSUInteger _flushInterval;
+    BOOL _enableABTestDesigner;
 }
 
 #if !defined(MIXPANEL_APP_EXTENSION)
 @property (nonatomic, assign) SCNetworkReachabilityRef reachability;
 @property (nonatomic, strong) CTTelephonyNetworkInfo *telephonyInfo;
+@property (nonatomic, strong) UILongPressGestureRecognizer *testDesignerGestureRecognizer;
```

Comment:
I might be missing something but what's the purpose of `testDesignerGestureRecognizer`?